THIRD-PARTY

```diff
@@ -0,0 +1,204 @@
+** reporting-kibana-plugin; version 1.0 --
```

Comment:
Add all dependencies and it's license to `THIRD-PARTY`

Reply:
refer to https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/master/THIRD-PARTY